cmd/access/node_builder/access_node_builder.go

```diff
@@ -1218,6 +1218,11 @@ func (builder *FlowAccessNodeBuilder) initPublicLibp2pNode(networkKey crypto.Pri
 		networkKey,
 		builder.SporkID,
 		&builder.FlowConfig.NetworkConfig.ResourceManagerConfig,
+		&p2pconfig.PeerManagerConfig{
+			ConnectionPruning: connection.PruningDisabled,
```

Comment:
Looks like this was already disabled, which is OK for now since there are only a few public nodes. Eventually we will need to have pruning enabled since there may be a large number of peers.

Reply:
https://github.com/onflow/flow-go/pull/4524/commits/9f96fd2c2821d494ad3352f74391dbd9c1ac86ee